### ISSUE TITLE
Update multiple AppImages at once

### ISSUE THREAD

**Issue by articlecat** [AppImage/AppImageUpdate#141]
It would be great if AppImageUpdate were able to update multiple AppImages at once. The process of individually updating each AppImage becomes time-consuming when there are a large number of AppImages on the system. A bulk update interface would significantly improve user experience for heavy AppImage users.

**Comment by probonopd** [comment]
Yes. It would be good if one could select e.g., a directory in addition to selecting a file.

(That being said, I personally never just bulk update everything. I only ever update an app if I see a real need to do so. Others may have other habits, though.)

**Comment by TheAssassin** [comment]
Please explain what user experience you expect for the UI. Right now we'd need a sort of "list dialog" from which we spawn the single updates or so.

Or would CLI support be sufficient? We could easily implement batch support there, it's probably just running the existing code in a loop.

**Comment by probonopd** [comment]
Ideally, the following would work:

* The user drags a bunch of AppImage files and/or folders that contain AppImages onto the AppImageUpdate icon, and they all will be updated (I know that unlike the Mac, Linux desktops typically can't handle this type of drag-and-drop well - but hey, one can dream...)
* The user opens AppImageUpdate, and instead of selecting an AppImage file in the file open dialog, selects multiple AppImage files and/or folders that contain AppImages
* The passes a bunch of AppImage files and/or folders that contain AppImages as arguments to the `appimageupdatetool` CLI tool
